Terasology/WildAnimalsGenome
# Register BreedingAlgorithm with trait

**Contains**
Changed the setSeedBasedProperty method call in the AnimalGeneticsSystem to pass the Breeding Algorithm as an argument after the changes made to genomes in https://github.com/Terasology/Genome/pull/10 .

**How to test**
- Use the `spawnPrefab deer` command to spawn multiple deer (atleast 2) in the world.
- Activate the deer by pressing `e` while looking at them and click activate mating
- Wait for the mating process to conclude and watch the console for feedback on the mating process

For best results, make sure https://github.com/Terasology/Genome/pull/15 is merged/in your workspace to avoid incorrect breeding failures. Certain issues with respect to the pathfinding still exist, however they are unrelated to this PR
